src/plugin.ts

```diff
@@ -82,14 +34,14 @@ export async function plugin(inputs: PluginInputs, env: Env) {
     } catch (e) {
       if (e instanceof CommanderError) {
         if (e.code !== "commander.unknownCommand") {
-          await context.logger.error(e.message);
+          context.logger.error(e.message);
         }
       } else {
-        await context.logger.error(e);
+        context.logger.error(`${e}`);
```

Comment:
Seems unusual to coerce to a string with this syntax.

Reply:
Typically we add a string yeah, avoid empty strings as much as possible. 

`context.logger.error("Error string", { e });`

no need to cast as Error if you just pass the `e` object

Reply:
Probably can just pass e object straight in instead of nesting inside another object.

Reply:
I think the type expects to get the error within an object and not the error straight in, at least with our current version of the logger. Could be changed later.